configure

```diff
@@ -0,0 +1,79 @@
+#!/bin/sh
+
+##==============================================================================
+##
+## configuredir
+## rootrelative
+## root
+##     Determine the root directory of the distribution (not necessarily the
+##     same as the current directory, since configure can be run anywhere).
+##
+##==============================================================================
+
+configuredir=`pwd`
+rootrelative=`dirname $0`
+cd $rootrelative
+root=`pwd`
+cd $configuredir
+
+
+##==============================================================================
+##
+## Product, Version, and Bilddate information:
+##
+##==============================================================================
+
+product="OMIScriptProvider"
+fullproduct="OMI Provider for scripting languages for Linux"
+
+# MI_MAJOR and other items are dependent on a rational version number
+# As a result, if version file can't be found, consider it fatal and
+# don't generate a bogus one
+
+if [ -f ../../omiscriptprovider.version ]; then
+    . ../../omiscriptprovider.version
+fi
+
+if [ -n "$OMI_SCRIPT_PROVIDER_BUILDVERSION_MAJOR" -a -n "$OMI_SCRIPT_PROVIDER_BUILDVERSION_MINOR" -a -n "$OMI_SCRIPT_PROVIDER_BUILDVERSION_PATCH" -a -n "$OMI_SCRIPT_PROVIDER_BUILDVERSION_BUILDNR" ]; then
+    major=$OMI_SCRIPT_PROVIDER_BUILDVERSION_MAJOR
+    minor=$OMI_SCRIPT_PROVIDER_BUILDVERSION_MINOR
+    revision=$OMI_SCRIPT_PROVIDER_BUILDVERSION_PATCH
+    patch_level=$OMI_SCRIPT_PROVIDER_BUILDVERSION_BUILDNR
+else
+    echo "*** UNABLE TO LOCATE FILE omiscriptprovider.version; USING VERSION 1.1.1.1 ***" >& 2
+    echo "*** Update build process to include omiscriptprovider.version from superproject! ***" >& 2
+
+    major=1
+    minor=1
+    revision=1
+    patch_level=1
+fi
+
+version="$major.$minor.$revision"
+version_with_release="$major.$minor.$revision-$patch_level"
+date=`date`
+
+
+##==============================================================================
+##
+## Create config.mak file
+##
+##==============================================================================
+
+fn=$configuredir/config.mak
```

Comment:
Not 100% sure where this is since it's dependent on where you're running from. In general, though, we try to build most files in a temporary directory (.../output for `omi`, `.../intermediate` for SCXCore, etc. I won't require that here, but extra credit if you put output files in different places than your source files.

Reply:
The output files will be placed under the output directory now.

Reply:
Is that true just for the configure output files, or the output from the `make` process as well?

Reply:
That is true for the configure output files and for the ```make release``` process which generates the deb and rpm packages. I'll update the PR to also include all the output files from the ```make``` process.

Reply:
Done.